src/components/user/login/login-form.js

```diff
@@ -0,0 +1,22 @@
+import React, { PureComponent } from 'react';
```

Comment:
This component is identical to _ src/components/user/register/register-form.js_ except for the async validate functionality. The extra layer of abstraction from _src/components/common/form/index.js_ doesn't seem necessary as there is no logic in the render function.

I suggest moving any logic in the `export default` statement to the _src/components/common/form/index.js_ abstraction. Refer to the reduxForm API documentation to see what options you have to make this code more DRY. There is an attribute in the options object called `form` which can be passed in to uniquely identify each form (https://redux-form.com/8.2.2/docs/api/props.md/#-code-form-string-code-)

Reply:
This is a really great suggestion. Thanks! I no longer need these 2 files. :)

Reply:
👍